### DISCUSSION TITLE
Logstash failed to start -Solved-

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#4364] [Unsupported Versions] [answered]
Hi,

I tried to start logstash as I was setting up winlogbeat but it failed to start. Someone please help.

![image](https://user-images.githubusercontent.com/62762611/120345214-e646dd00-c317-11eb-9bbf-658a83900d6f.png)

**Comment by Jackson-Pollock** [comment, marked as answer]
Please suggest

**Reply by Jackson-Pollock** [reply]
solved

**Reply by weslambert** [reply]
Could you please clarify your solution before marking as the answer, so that others can benefit?
